src/lib.rs

```diff
@@ -23,6 +23,29 @@ pub struct ParseConfig {
     pub custom_parsers: Vec<Box<dyn crate::parsers::StructuredLogParser>>,
 }
 
+fn maybe_remove_suffix(frames: &mut Vec<FrameSummary>) {
```

Comment:
I feel like you'd want to name this something more specific like `maybe_remove_convert_frame_suffixes`, or make the function more general by being able to pass in a list of target frames instead of hard coding it

Reply:
Thanks, https://github.com/ezyang/tlparse/pull/30